packages/forma-36-website/src/pages/components/example/index.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Example Component'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-formlabel--default'
+---
+
+Add a description about the component and its function in the user interface here
+E.g: TextLinks communicate actions and highlight linked resources to users.
+
+Add an example of the component like this:
+
+```jsx
+<React.Fragment>
+  <Button buttonType="primary">Primary</Button>
+</React.Fragment>
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Button types](#button-types)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
+
+## Best practices <a name="best-practices" />
+
+Add best practices for usage in the UI, think about copy and accessibility or general UX guidelines
```

Comment:
Include best practices for how the component should be used in the context of the overall experience.